scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -838,7 +843,7 @@ task AddIsLoadedColumn {
 
     # set is_loaded to true if there is a corresponding pet table partition with rows for that sample_id
     bq --location=US --project_id=~{project_id} query --format=csv --use_legacy_sql=false \
-    "UPDATE ~{dataset_name}.sample_info SET is_loaded = true WHERE sample_id IN (SELECT CAST(partition_id AS INT64) from ~{dataset_name}.INFORMATION_SCHEMA.PARTITIONS WHERE partition_id != '__UNPARTITIONED__' AND total_logical_bytes > 0 AND table_name LIKE \"pet_%\")"
+    "UPDATE ~{dataset_name}.sample_info SET is_loaded = true WHERE sample_id IN (SELECT CAST(partition_id AS INT64) from ~{dataset_name}.INFORMATION_SCHEMA.PARTITIONS WHERE partition_id in ('~{sep="\',\'" gvs_id_array}') AND total_logical_bytes > 0 AND table_name LIKE \"pet_%\")"
```

Comment:
this made me wonder if there would be a benefit to checking and validating that there are no samples with a partition_id of '__UNPARTITIONED__'....

Reply:
I think that could be a good check. but i'm wondering where/when to do it? I guess we could do it here and fail if there is data in unpartitioned.

Reply:
yeah I'm not sure where the check could go---maybe just print out a warning in the mean time? I dont think it's necessary to this pr though and would be fine as a follow on ticket